frontend/app/enrollment/components/enrollment-details-view.tsx

```diff
@@ -40,7 +40,8 @@ export default function EnrollmentDetailsView() {
                     <div className='font-bold'>Service:</div>
                     <div className='col-span-3'>{serviceRequest?.code?.coding?.[0].display || "Unknown"}</div>
                     <div className='font-bold'>Care Path:</div>
-                    <div className='col-span-3'>{taskCondition?.code?.coding?.[0].display || "Unknown"}</div>
+                    {/* TODO: Check whether to prefer code.text or code.coding.display - NullFlavour hurts in coding display */}
```

Comment:
both?

Reply:
Yeah, can do. But it's a but ugly as they do contain values, like `other` that need to be checked/excluded